modules/sql-engine/src/integrationTest/sql/order/test_order_variable_size_payload.test_ignore

```diff
@@ -206,23 +217,7 @@ statement ok
 CREATE TABLE test6 (i1 INT, s1 VARCHAR, i2 int, s2 VARCHAR)
 
 statement ok
-INSERT INTO test6 VALUES
-(6, '0reallylongstring1', 3, '1reallylongstring8'),
-(6, '0reallylongstring1', 3, '1reallylongstring7'),
-(6, '0reallylongstring1', 4, '1reallylongstring8'),
-(6, '0reallylongstring1', 4, '1reallylongstring7'),
-(6, '0reallylongstring2', 3, '1reallylongstring8'),
-(6, '0reallylongstring2', 3, '1reallylongstring7'),
-(6, '0reallylongstring2', 4, '1reallylongstring8'),
-(6, '0reallylongstring2', 4, '1reallylongstring7'),
-(5, '0reallylongstring1', 3, '1reallylongstring8'),
-(5, '0reallylongstring1', 3, '1reallylongstring7'),
-(5, '0reallylongstring1', 4, '1reallylongstring8'),
-(5, '0reallylongstring1', 4, '1reallylongstring7'),
-(5, '0reallylongstring2', 3, '1reallylongstring8'),
-(5, '0reallylongstring2', 3, '1reallylongstring7'),
-(5, '0reallylongstring2', 4, '1reallylongstring8'),
-(5, '0reallylongstring2', 4, '1reallylongstring7')
+INSERT INTO test6 VALUES (6, '0reallylongstring1', 3, '1reallylongstring8'), (6, '0reallylongstring1', 3, '1reallylongstring7'),(6, '0reallylongstring1', 4, '1reallylongstring8'),(6, '0reallylongstring1', 4, '1reallylongstring7'),(6, '0reallylongstring2', 3, '1reallylongstring8'),(6, '0reallylongstring2', 3, '1reallylongstring7'),(6, '0reallylongstring2', 4, '1reallylongstring8'),(6, '0reallylongstring2', 4, '1reallylongstring7'),(5, '0reallylongstring1', 3, '1reallylongstring8'),(5, '0reallylongstring1', 3, '1reallylongstring7'),(5, '0reallylongstring1', 4, '1reallylongstring8'),(5, '0reallylongstring1', 4, '1reallylongstring7'),(5, '0reallylongstring2', 3, '1reallylongstring8'),(5, '0reallylongstring2', 3, '1reallylongstring7'),(5, '0reallylongstring2', 4, '1reallylongstring8'),(5, '0reallylongstring2', 4, '1reallylongstring7')
```

Comment:
What reason for the change? We don't support multilines in SQL logic tests?

Reply:
Seems so, for me test fails with the following
```
org.opentest4j.AssertionFailedError: Not expected result at: (test_order_variable_size_payload.test:27). Statement: INSERT INTO test0 VALUES ('Shipping and Receiving Supervisor', 'Ackerman'),

	at org.apache.ignite.internal.sql.sqllogic.Statement.execute(Statement.java:112)
	at org.apache.ignite.internal.sql.sqllogic.SqlScriptRunner.run(SqlScriptRunner.java:70)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.apache.ignite.sql.SqlException: IGN-SQL-3 TraceId:ba640ebb-929e-43c5-84da-fbd9d2b39cd4 Failed to parse query: Encountered "," at line 1, column 75
```

Reply:
looks like this, for me test fails with the following
```
org.opentest4j.AssertionFailedError: Not expected result at: (test_order_variable_size_payload.test:144). Statement: INSERT INTO test6 VALUES (6, '0reallylongstring1', 3, '1reallylongstring8'),

	at org.apache.ignite.internal.sql.sqllogic.Statement.execute(Statement.java:112)
	at org.apache.ignite.internal.sql.sqllogic.SqlScriptRunner.run(SqlScriptRunner.java:70)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.apache.ignite.sql.SqlException: IGN-SQL-3 TraceId:eef5308c-7505-416c-b779-31ca06190e87 Failed to parse query: Encountered "," at line 1, column 76
```